queries/rust/highlights.scm

```diff
@@ -148,6 +148,12 @@
   (block_comment)
 ] @comment @spell
 
+((line_comment) @comment.documentation
+  (#match? @comment.documentation "^//[/!][^/].*"))
+
+((block_comment) @comment.documentation
+  (#match? @comment.documentation "^/\\*[\\*!][^\\*].*\\*/"))
+
```

Comment:
This doesn't match empty doc comments, like just `//!` or `///`, would like advice on how to fix that